gateway/src/cluster/builder.rs

```diff
@@ -3,10 +3,15 @@ use super::{
     event::Events,
     r#impl::{Cluster, ClusterStartError},
     scheme::ShardScheme,
-    ClusterStartErrorType,
 };
+#[cfg(any(
+    feature = "native",
+    feature = "rustls-native-roots",
+    feature = "rustls-webpki-roots"
+))]
+use crate::shard::tls::TlsContainer;
```

Comment:
just a style question, should we have this import in the list of imports or below it for clarity?